.circleci/config.yml

```diff
@@ -66,7 +66,7 @@ jobs:
       - run:
           name: Build Docker image
           no_output_timeout: 30m
-          command: docker build -f ./docker/Dockerfile -t $IMAGE_NAME:latest --build-arg GIT_HASH=$CIRCLE_SHA1 .
+          command: docker build -f ./docker/Dockerfile -t $IMAGE_NAME:latest
```

Comment:
Maybe I'm missing where this happens, but are you sure we have git initialized at this point? Otherwise there's no `.git` folder for docker to pick up